bin/kano-updater

```diff
@@ -186,6 +186,7 @@ def main():
     if gui_mode:
         # Hide the home button only if we are on Desktop mode
         run_bg('systemctl --user status kano-desktop && /usr/bin/kano-home-button-visible no')
+        run_bg('kano-stop-splash dashboard')
```

Comment:
Yeah, looks okay, but I think we'll need a few more things than that.

 * Use the new better "sudo systemctl stop kano-boot-splash" - or new equivalent name
 * Move the systemd service that stops the splash to a later stage, as I noticed weird UI behaviour on recovery
 * Probably disable plymouth, which seems to try to do things on recovery